wrapanapi/rhevm.py

```diff
@@ -96,7 +96,8 @@ class RHEVMSystem(WrapanapiAPIBase):
     def __init__(self, hostname, username, password, **kwargs):
         # generate URL from hostname
         super(RHEVMSystem, self).__init__(kwargs)
-        url_component = 'ovirt-engine/api' if float(kwargs['version']) >= 4.0 else 'api'
+        rhv_gt_4 = float(kwargs['version']) >= 4.0
```

Comment:
No logical change here, just pulling the version query into a separate boolean for re-use.